sample-inventories/inventory-a/group_vars/reveal_web/vars.yml

```diff
@@ -0,0 +1,206 @@
+  EXPRESS_FRONTEND_OPENSRP_CALLBACK_URL: "https://{{ deployed_site_name }}/fe/oauth/callback/opensrp"
+  EXPRESS_OPENSRP_CLIENT_ID: "{{ opensrp_client_id }}"
+  EXPRESS_OPENSRP_CLIENT_SECRET: "{{ vault_express_opensrp_client_secret }}"
+  EXPRESS_FRONTEND_LOGIN_URL: "/fe/login"
+
+# nginx
+nginx_install_only: true
+nginx_version: "1.18.*"
+reveal_nginx_site_names:
+  - "{{ deployed_site_name }}"
+reveal_nginx_site_name: "{{ reveal_nginx_site_names[0] }}"
+reveal_nginx_http_site_name: "{{ deployed_site_name }}-http"
+reveal_nginx_https_site_name: "{{ deployed_site_name }}-https"
+reveal_nginx_access_logs:
+  - name: "timed_combined"
+    format: "'$http_x_forwarded_for - $remote_user [$time_local]  \"$request\" $status $body_bytes_sent \"$http_referer\" \"$http_user_agent\" $request_time $upstream_response_time $gzip_ratio $request_length'"
+    options: null
+    filename: "access.log"
+reveal_nginx_enabled_sites:
+  - "{{ reveal_nginx_http_site_name }}"
+  - "{{ reveal_nginx_https_site_name }}"
+reveal_nginx_ssl_dir: /etc/nginx/ssl/{{ deployed_site_name }}
+reveal_nginx_sites:
+  - server:
+      name: "{{ reveal_nginx_http_site_name }}"
+      listen:
+        - "80"
+        - "[::]:80"
+      server_name: "{{ reveal_nginx_site_names|join(' ') }}"
+      access_log: "{{ reveal_nginx_log_path }}/{{ deployed_site_name }}-access.log timed_combined"
+      error_log: "{{ reveal_nginx_log_path }}/{{ deployed_site_name }}-error.log"
+      rewrite: "^(.*) https://{{ reveal_nginx_site_name }}$1 permanent"
+      ssl:
+        enabled: false
+  - server:
+      name: "{{ reveal_nginx_https_site_name }}"
+      listen:
+        - "443 ssl"
+        - "[::]:443 ssl"
+      server_name: "{{ reveal_nginx_site_names|join(' ') }}"
+      access_log: "{{ reveal_nginx_log_path }}/{{ deployed_site_name }}-ssl-access.log timed_combined"
+      error_log: "{{ reveal_nginx_log_path }}/{{ deployed_site_name }}-ssl-error.log"
+      ssl:
+        add_ssl_directive: false
+        access_log_format: "timed_combined"
+        enabled: true
+        remote_src: true
+        create_symlink: true
+        cert: "fullchain.pem"
+        key: "privkey.pem"
+        src_dir: "/etc/letsencrypt/live/{{ deployed_site_name }}"
+        conf: "{{ deployed_site_name }}.conf"
+      # do not cache the service worker
+      location1:
+        name: "/service-worker.js"
+        add_header: 'Cache-Control "no-cache"'
+        proxy_cache_bypass: "$http_pragma"
+        proxy_cache_revalidate: "on"
+        expires: "off"
+        access_log: "off"
+      location2:
+        name: "/"
+        autoindex: "off"
+        charset: "UTF-8"
+        proxy_pass: http://localhost:3000
+        proxy_http_version: 1.1
+        proxy_set_header:
+          - "Upgrade $http_upgrade"
+          - "Connection 'upgrade'"
+          - "Host $host"
+          - "X-Forwarded-Proto https"
+        proxy_cache_bypass: $http_upgrade
+reveal_nginx_monit_protection: false
```

Comment:
Should the default behavior be to disable Monit?

Reply:
I tend to build images than run ansible against an existing setup, you normally would enable this on startup hence the default value has been false.

Reply:
cool